apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.ts

```diff
@@ -510,6 +542,30 @@ export class EWayBillComponent implements OnInit, OnDestroy {
             this.getAllFilteredInvoice();
         }
     }
+    /**
+     * Aside pane toggle fixed class
+     *
+     *
+     * @memberof GstComponent
+     */
+     public toggleBodyClass(): void {
+        if (this.asideGstSidebarMenuState === 'in') {
+            document.querySelector('body').classList.add('gst-sidebar-open');
+        } else {
+            document.querySelector('body').classList.remove('gst-sidebar-open');
+        }
+    }
+    /**
+      * This will toggle the settings popup
```

Comment:
Correct JS doc for this method it will toggle the GST popup and not settings popup